Cereal/CerealDecoder.swift

```diff
@@ -52,11 +87,12 @@ public struct CerealDecoder {
     - returns:      The instantiated object, or nil if no object was at the specified key.
     */
     public func decode<DecodedType: CerealRepresentable>(key: String) throws -> DecodedType? {
-        guard let data = items[key] else {
+        guard let data = self.itemForKey(key) else {
```

Comment:
(or any of the other newly altered decode methods? I prefer not to have `self` if its not necessary)